src/Athena/Views/Home/Main_Page.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@{
+    ViewData["Title"] = "Main_Page";
+}
+<br /><br /><br /><br /><br />
+
+<img src="~/images/UT-color-vert-trans.gif" class="responsive-img left" style="height:10em; max-width:100%" />
+<br /><br /><br /><br /><br /><br /><br /><br /><br /><br />
+
+
+    <h4>Degree Progress:</h4>
+    <br />
+    <div id="progress-bar" class="all-rounded" style="height:2em;width:10em">
```

Comment:
Use the [grid system](http://materializecss.com/grid.html) for size and layout control instead of manually specifying position or size.